Scripts/checkout-submodule.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -euo pipefail
+
+git submodule foreach git submodule update --init --recursive
```

Comment:
How is this different (in practice) from:
```suggestion
git submodule update --init --recursive
```